lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/hmrcKbvCri/ContractTest.java

```diff
@@ -532,8 +520,6 @@ public RequestResponsePact validRequestReturnsIssuedCredentialWithCi(
                 .given("VC birthDate is 1932-02-25")
                 .given("VC evidence verificationScore is 0")
                 .given("VC evidence txn is dummyTxn")
-                .given("VC contains a socialSecurityRecord")
-                .given("VC personalNumber is AA000003D")
```

Comment:
Not sure why you've removed `personalNumber` the value is variable so we should specify what we're expecting. I'm also still not clear on how the provider side test writer will know what types of correct and incorrect questions we are expecting in the VC.
Should we have a chat about these `given()` functions?